package/scripts/httpfs.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+import os
+from resource_management import *
+
+import logging
+
+logger = logging.getLogger()
+
+class HTTPFS(Script):
+  def install(self, env):
+    import params
+    env.set_params(params)
+    self.install_packages(env)
+
+
+  def configure(self, env):
+    import params
+    env.set_params(params)
+
+    Directory(params.app_conf_dir)
+
+    File(params.app_conf_dir + '/httpfs-site.xml',
+         content = Template('httpfs-site.xml.j2'),
+         owner = params.user,
+         group = params.group)
+
+    File(params.app_conf_dir + '/httpfs-env.sh',
+         content = StaticFile('httpfs-env.sh'),
```

Comment:
Yes, I was talking about the config files. I understand your point. However I'm not seeing them included anywhere, are they already in the package ?